.Documentation/articles/step-by-step-guides/hello-creator.md

```diff
@@ -0,0 +1,121 @@
+# Hello Creator – a 5-step guide to a basic training application
+
+You have successfully downloaded and imported the Innoactive Creator. This guide will help you to create your first basic training application in 5 steps. During the steps, we introduce you to the basic User interface of the Unity Editor and the Innoactive Creator and explain basic concepts of a successful creation of a training application using the Creator.
+
+## Step 1: Create Course
+
+In case you are new to unity, you see an empty unity project, which usually includes a **scene** window, a **hierarchy** of 3D objects of your scene which is currently empty and only contains a Main Camera and Directional Light, the Inspector (might also be empty currently), and on the lower end of the unity window, a **project hierarchy and console** window (see figure 1).
```

Comment:
```suggestion
In case you are new to Unity, you see an empty Unity project, which usually includes a **scene** window, a **hierarchy** of 3D objects of your scene which is currently empty and only contains a `Main Camera` and `Directional Light`, the Inspector (might also be empty currently), and on the lower end of the Unity window, a **project hierarchy and console** window (see figure 1).
```